ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/button/v1/button/button.html

```diff
@@ -0,0 +1,34 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2019 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<sly data-sly-use.renderer="${'button.js'}"
+  data-sly-use.button="com.adobe.cq.forms.core.components.models.form.Button"
+  data-sly-use.component="com.adobe.cq.wcm.core.components.models.Component"
+  data-sly-use.clientlib="${'/libs/granite/sightly/templates/clientlib.html'}"
+  data-sly-use.shortDescription="${renderer.shortDescriptionPath}"
+  data-sly-use.longDescription="${renderer.longDescriptionPath}"
+  data-sly-use.questionMark="${renderer.questionMarkPath}"></sly>
+<button 
+  type="${button.buttonType}"
+  value="${button.value}"
+  id="${component.id}"
+  class="cmp-button"
+  data-cmp-is="adaptiveFormButton"
+  aria-label="${button.label.value}">
+<span data-sly-test="${button.label.value}" class="cmp-button__text">${button.label.value}</span>
```

Comment:
cp-button__label. Can you check the wcm button and align the markup with that
cc @nit23uec @rismehta